app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+
+const {getTopics} = require('./controllers')
+
+const app = express();
+app.use(express.json())
```

Comment:
We should not have this line in here as it is not needed, we could add it back later 🙂

Reply:
Done, thank you very much :)